packages/collector/test/tracing/database/db2/test.js

```diff
@@ -14,22 +15,30 @@ const ProcessControls = require('../../../test_util/ProcessControls');
 const globalAgent = require('../../../globalAgent');
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
-const DB_LOCAL_CONN_STR = 'HOSTNAME=localhost;UID=node;PWD=<redacted>;PORT=58885;PROTOCOL=TCPIP';
-const DB_REMOTE_CONN_STR = testUtils.isCI()
-  ? process.env.DB2_CONNECTION_STR.replace(/PWD=.*?(?=;)/, 'PWD=<redacted>')
-  : null;
 
-const CONN_STR = DB_REMOTE_CONN_STR || DB_LOCAL_CONN_STR;
-let DB2_NAME;
+if (testUtils.isCI() && !process.env.DB2_CONNECTION_STR) {
+  throw new Error(
+    'No connection string for IBM DB2, please make sure the environment variable DB2_CONNECTION_STR is set.'
+  );
+}
+
+const DB2_CONN_STR =
+  process.env.DB2_CONNECTION_STR || 'HOSTNAME=localhost;UID=node;PWD=nodepw;PORT=58885;PROTOCOL=TCPIP';
+let DB2_CONN_STR_ALTERNATIVE;
+let EXPECTED_DB2_CONN_STR;
+let EXPECTED_DB2_CONN_STR_ALTERNATIVE;
+const DB2_HOSTNAME = /HOSTNAME=([^;]+)/.exec(DB2_CONN_STR)[1];
+
+const DB2_DATABASE_NAME = 'nodedb';
```

Comment:
I don't remember, to be honest. I think I mainly wanted to remove the function because it returned the same hard coded string for each call. I can bring the comment back.